src/rp2_common/hardware_gpio/include/hardware/gpio.h

```diff
@@ -1403,12 +1403,12 @@ extern void gpio_debug_pins_init(void);
 #endif
 
 
-// PICO_CONFIG: PICO_DEBUG_PIN_BASE, First pin to use for debug output (if enabled), min=0, max=28, default=19, group=hardware_gpio
+// PICO_CONFIG: PICO_DEBUG_PIN_BASE, First pin to use for debug output (if enabled), min=0, max=NUM_BANK0_GPIOS - 1, default=19, group=hardware_gpio
 #ifndef PICO_DEBUG_PIN_BASE
 #define PICO_DEBUG_PIN_BASE 19u
 #endif
 
-// PICO_CONFIG: PICO_DEBUG_PIN_COUNT, Number of pins to use for debug output (if enabled), min=1, max=28, default=3, group=hardware_gpio
+// PICO_CONFIG: PICO_DEBUG_PIN_COUNT, Number of pins to use for debug output (if enabled), min=1, max=NUM_BANK0_GPIOS, default=3, group=hardware_gpio
```

Comment:
this is 32 still (though actually really BASE + COUNT <=32), but i don't think we care atm